components/cdc/src/endpoint.rs

```
@@ -1203,8 +1204,8 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Runnable for Endpoint<T> {
                 cb,
             } => {
                 debug!("downstream was initialized"; "downstream_id" => ?downstream_id);
-                downstream_state
-                    .compare_and_swap(DownstreamState::Uninitialized, DownstreamState::Normal);
+                let _ = downstream_state
```

Is it safe to ignore error? @5kbpers @overvenus

---

yes

---

If so, why not just use `store`?

---

Could you unwrap it?

---

Because it could be `Stopped` state

---

Is it safe to transit state from `Stopped` to `Normal`?

---

No, so we should not unwrap it here